pyrokinetics/gk_code/GKOutputReader.py

```diff
@@ -50,15 +51,20 @@ class GKOutputReader(Reader):
         kx          1D array of floats
         ky          1D array of floats
         theta       1D array of floats
-        energy      1D array of floats
+        energy_dim  1D array of floats
         pitch       1D array of floats
         moment      ["particle", "energy", "momentum"]
         field       ["phi", "apar", "bpar"] (the number appearing depends on nfield)
         species     list of species names (e.g. "electron", "ion1", "deuterium", etc)
 
     data_vars
-        fields      (field, theta, kx, ky, time) complex array, may be zeros
-        fluxes      (species, moment, field, ky, time) float array, may be zeros
+        phi                    (theta, kx, ky, time) complex array, units [qref / tref * lref / rhoref], may be zeros
+        apar                   (theta, kx, ky, time) complex array, units [1 / bref * lref / rhoref **2], may be zeros
+        bpar                   (theta, kx, ky, time) complex array, units [1 / bref * lref / rhoref], may be zeros
```

Comment:
In GS2 bpar use B(theta) rather than B_ref to normalise Bpar. This might complicate things. Probably the easiest would be to adjust for this when reading in (e.g. scale by Bmag)?